Homework/Van_Le/Duplicate.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+
+namespace ContainDuplicate
+{
+    public static class Duplicate
+    {
+        public static bool ContainsDuplicate(int[] nums)
+        {
+            Dictionary<int, int> myMap = new Dictionary<int, int> ();
```

Comment:
use better name?

Reply:
which one?

Reply:
please review it I have changed the name